v3/src/components/case-table/use-columns.tsx

```diff
@@ -52,6 +54,24 @@ export function renderValue(str = "", num = NaN, attr?: IAttribute, key?: number
     if (formatter) str = formatter(num)
   }
 
+  // dates
+  // Note that CODAP v2 formats dates in the case table ONLY if the user explicitly specified type as "date".
+  // Dates are not interpreted as dates and formatted by default.
+  if (userType === "date" && str !== "") {
+    const date = parseDate(str, true)
+    if (date) {
+      // TODO: add precision support for date formatting
+      const formattedDate = formatDate(date, DatePrecision.None)
+      return {
+        value: str,
+        content: <span className="cell-span" key={key}>{formattedDate || str}</span>
```

Comment:
For consistency, shouldn't this be:
```suggestion
        content: <span className="cell-span" key={key}>{formattedDate || `"${str}"`}</span>
```